### DISCUSSION TITLE
what's an area of interest as it relates to building on kaia?

### DISCUSSION THREAD

**Discussion by PaulElisha** [kaiachain/kaia-dapp-mono#181] [General]
when it comes to building protocols, platforms, are you more invested in defi, and what project would you expect on kaia?

**Comment by zxstim** [comment]
i personally only like defi and believe that defi + stablecoin is the pathway to mass adoption

but i am pretty open overall. i want to see more artists selling arts, building communities, and pushing a new movement to get more people onchain

we seriously need more open source infra, like block explorer, indexer softwares so projects can have alternatives.

...and more. some ideas i listed here: https://github.com/kaiachain/kaia-dapp-mono/blob/main/builder-program/ideas.md
